src/index.html

```diff
@@ -2,16 +2,98 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="input-body">
+        <legend>Personal infomation</legend>
+        <label class="input-block">Surname:
+          <input type="Surname">
+        </label>
+        <label class="input-block">Name:
+          <input type="Name">
+        </label>
+        <label class="input-block">How old are You?
+          <input 
+          type="number" 
+          min="1" 
+          max="100"
+          value="12"
+          >
```

Comment:
fix all your indentations according to checklist 
![image](https://user-images.githubusercontent.com/59209337/222356069-66e3e3d8-79d4-4b4f-b284-595d42aabef6.png)

Reply:
Not resolved. Check an example of good code (previous comment)